src/scripts/pbs_status.py

```diff
@@ -234,12 +234,11 @@ def qstat(jobid=""):
     starttime = time.time()
     log("Starting qstat.")
     command = (qstat_bin, '-f')
-    if config.get('pbs_pro').lower() == 'yes':
-        command += ('-x',)  # also query for detailed output of finished jobs
-    else:
+    pbs_pro = config.get('pbs_pro').lower() == 'yes'
```

Comment:
Is `'pbs_pro'` guaranteed to be in `config`?

The `config.get('pbs_pro')` construction (instead of `config['pbs_pro']`) makes me think it's ok for it not to be, but the `.lower()` of course would not on the `None` returned if the key is missing.

Reply:
Yup, we set a default when creating `config`: https://github.com/opensciencegrid/BLAH/blob/331415529e8afe8819dddd4b9f40d7a80da11d81/src/scripts/pbs_status.py#L553-L555